### PR TITLE
Make new frontend (Angular) default frontend instead of erb

### PR DESCRIPTION
**Pull request by nakhbari**
Fixes: #767 

Load the Angular app by default, and launch with ERB client when the `ERB_CLIENT` env key is set.

One drawback here is the lack of login implementation in the Angular app. This switch should motivate us to get these features done.

### CODE REVIEW

File: fastlane_app.rb [hunk 1]
```diff
@@ -47,24 +47,24 @@ class FastlaneApp < Sinatra::Base
...
     # display of real-time output
     set(:server, "thin")
-    if ENV["WEB_APP"]
-      # Anything except a data route
-      get %r{/(?!data.*).*} do
-        # Use Angular Web App instead
-        send_file(File.join("public", ".dist", "index.html"))
-      end
-    else
+    if ENV["ERB_CLIENT"]
```
Comment (taquitos) [inline fastlane_app.rb]: let's make it "FASTLANE_CI_ERB_CLIENT"

Comment (nakhbari) [reply fastlane_app.rb]: done

**Comment by taquitos** [comment]
🚢

**Comment by KrauseFx** [comment]
Don't forget to delete the branches 👍

**Pull request merged**